assemble/bin/accumulo-cluster

```diff
@@ -299,9 +299,9 @@ function stop_all() {
 
 function stop_here() {
   # Determine hostname without errors to user
-  hosts_to_check=($(hostname -a 2> /dev/null | head -1) $(hostname -f))
+  hosts_to_check="$(hostname -a 2> /dev/null | head -1) $(hostname -f)"
 
-  if egrep -q localhost\|127.0.0.1 "${conf}/tservers"; then
+  if grep -E -q localhost\|127.0.0.1 "${conf}/tservers"; then
```

Comment:
The pattern here would make more sense quoted, rather than escaped, and it looks like the dots aren't being matched properly. Here's a fix:

```suggestion
  if grep -Eq 'localhost|127[.]0[.]0[.]1' "${conf}/tservers"; then
```